conf/application.conf

```diff
@@ -96,6 +96,11 @@ tracking-consent-frontend {
   gtm.container = "d"
 }
 
+timemachine{
+    enabled = true
+    date = "now"
+}
+
```

Comment:
can you also update environment app-configs so
1. app-config-base to have timemachine.enabled defaulted to false
2. app-config-qa and app-config-staging timemachine enabled with date "now"
3. app-config-production to have defaulted false (a second guard).

Reply:
FYI this will need to be done for both penalties and appeals